src/main/java/org/spongepowered/api/world/generation/config/NoiseGeneratorConfig.java

```diff
@@ -76,6 +82,8 @@ static Builder builder() {
 
     boolean noodleCaves();
 
+    boolean legacyRandomSource();
```

Comment:
Is there any point to expose this?

Reply:
From what I can see e.g. nether and end are using this but overworld does not.